docs/CppInteroperability/GettingStartedWithC++Interop.md

```diff
@@ -75,9 +66,28 @@ int cxxFunction(int n);
 int cxxFunction(int n) {
     return n;
 }
+```
 
 ```
+// In main.swift
+
+import Foundation
+import CxxTest
+
+print("Hello, World! \(cxxFunction(7))")
+
+```
+
+## Adding C++ to an Xcode project
+In your xcode project, follow the steps [Creating a Module to contain your C++ source code](#creating-a-module-to-contain-your-c-source-code) in your project directory
+
+Add the C++ module to the include path and enable C++ interop:
+- Navigate to your project directory
+- In `Project` navigate to `Build Settings` -> `Swift Compiler`
+- Under `Custom Flags` -> `Other Swift Flags` add`-Xfrontend -enable-experimental-cxx-interop`. If you cannot find `Other Swift Flags`, please ensure all build settings have been shown.
+- Under `Swift Compiler - Search Paths` -> `Import Paths` add your search path to the C++ module (i.e, `$(SRCROOT)/CxxTest`). 
 
+This should now allow you to import your C++ Module into any `.swift` file.
```

Comment:
I think we can phrase this a little bit better. We can only import the C++ module from Swift files that belong to modules that declare a dependency on the C++ module, not just any Swift file, right?